pkg/operator/ceph/cr_manager.go

```diff
@@ -67,3 +104,42 @@ func (o *Operator) startManager(namespaceToWatch string, context context.Context
 		return
 	}
 }
+
+// Let's be gentle with the cache memory consumption
+// This restricts the cache's size to the specific objects in their respective namespaces we watch
+// This is governed by namespaceToWatch which is populated earlier, if namespaceToWatch is "" then we will do on all namespaces
+func getNewManagerCache(namespaceToWatch string) cache.NewCacheFunc {
+	namespaceSelector := fields.Set{"metadata.namespace": namespaceToWatch}.AsSelector()
+	return cache.BuilderWithOptions(
+		cache.Options{
+			SelectorsByObject: cache.SelectorsByObject{
+				&cephv1.CephCluster{}:                     {},
```

Comment:
If I read the doc right, this would mean that no fields of the CephCluster and objects with value `{}` would be cached.
And in others only the namespace field.

https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.5/pkg/cache#Options

Reply:
Are the reconciler GETs getting empty structs?